_posts/2020-01-21-hhvm-4.41.markdown

```diff
@@ -0,0 +1,40 @@
+---
+title: "HHVM 4.41"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.41 is released! This release marks the end of support for 4.34;
+HHVM 4.35&ndash;4.40 remain supported, as do the 4.8 and 4.32 LTS releases.
+
+# Highlights
+
+- A new type alias, `HH\enumname<TEnum>`, represents the name of the enum type
+  `TEnum` (similarly to `classname<TClass>`).
+- A new function, `HH\type_structure_for_alias`, returns the `TypeStructure` of
+  the specified type alias (similarly to `HH\type_structure` for type constants
+  in classes).
```

Comment:
not sure there's a good place to mention it, but I'm hoping that these will become autoimported shortly.

Reply:
Probably makes the most sense to mention it when it happens.